Documentation/gettingstarted/servicemesh/tls-termination.rst

```diff
@@ -0,0 +1,110 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_ingress_tls:
+
+************************************
+Ingress example with TLS termination
+************************************
+
+This example builds on the HTTP and gRPC ingress examples, adding TLS
+termination.
+
+Create TLS certificate and private key
```

Comment:
Oh, also do you think we could do a `.. tabs::` for the cert manager vs. this guide, so that only this section for configuring the certs is tabbed? Then the rest of the guide can be all in one page, shared between the two.

Reply:
good point :100: